docs/source/apps/hpc.md

```diff
@@ -0,0 +1,33 @@
+# HPC integration
```

Comment:
Feels like this is just a summary of the notebook, but no new information. I will merge them.

Reply:
Yes, I combine then.